index.html

```diff
@@ -143,10 +143,49 @@ <h3>
           </li>
           <li>Run the following steps [=in parallel=]:
             <ol>
+              <li>Let |allPorts:list&lt;SerialPort&gt;| be an empty [=list=].
+              </li>
+              <li>[=list/For each=] available Bluetooth device:
```

Comment:
To avoid confusion about whether "available" implies scanning do you want to say "paired" here instead?

Reply:
:pear: sgtm